source/interprocedural_analyses/taint/taintConfiguration.ml

```diff
@@ -489,12 +489,24 @@ module Heap = struct
   let default =
     let sources =
       List.map
-        ~f:(fun name -> { AnnotationParser.name; kind = Named })
+        ~f:(fun name ->
+          {
+            AnnotationParser.name;
+            kind = Named;
+            taint_config_position = "";
+            taint_config_path = PyrePath.current_working_directory ();
```

Comment:
1. Why do we need to store a position into `AnnotationParser.t`?
2. Could we use a proper type (i.e something like `Location.t`) to represent locations? It's also odd that the position is a string?